catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/EntityResourceTest.java

```diff
@@ -1029,4 +1037,20 @@ private void printEntities(ResultList<T> list) {
     list.getData().forEach(e -> LOG.info("{} {}", entityClass, getEntityInterface(e).getFullyQualifiedName()));
     LOG.info("before {} after {} ", list.getPaging().getBefore(), list.getPaging().getAfter());
   }
+
+  /**
+   * Given a list of properties of an Entity (e.g., List<Column> or List<MlFeature> and
+   * a function that validate the elements of T, validate lists
+   */
+  public <P> void assertListProperty(List<P> expected, List<P> actual, BiConsumer<P, P> validate) throws HttpResponseException {
```

Comment:
I added this utility here that might come in handy when running assertions on property lists. It allows us to just create the function checking the assertions and handles the boilerplate for us.

It might be useful in DBT and Table tests as well